src/containers/CallIn/ThankYou/ThankYou.js

```diff
@@ -63,6 +64,34 @@ export class ThankYou extends Component {
         const trackingToken = getUrlParameter(params, 't') || undefined;
         const callerId = getUrlParameter(params, 'c') || undefined;
         this.removeTrackingGetArgs();
+        if (!calledState || !calledNumber) {
+            if (!calledState) {
```

Comment:
Missing calledState would through a "Cannot Read Property .toLowerCase() of undefined" error if calledState was not provided. https://sentry.io/organizations/ben-boral/issues/2381416294/?project=1462422&query=is%3Aunresolved